mokojin/GetCharacters.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  GetCharacters.swift
+//  mokojin
+//
+//  Created by Yonatan Bergman on 3/23/15.
+//  Copyright (c) 2015 iicninjas. All rights reserved.
+//
+
+import Foundation
+import Parse
+
+class GetCharacters {
+    func get(callback: ([Character])->()){
+        PFCloud.callFunctionInBackground("getCharacters", withParameters: nil) { (rawResponse, err) -> Void in
```

Comment:
@yonbergman maybe use `()` here instead of `Void`? to match the definition of the `callback` parameter. Also in the other "get" operations..